boards/nordic/nrf5340_audio_dk/Kconfig.defconfig

```diff
@@ -65,3 +70,4 @@ endif # BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP || BOARD_NRF5340_AUDIO_DK_NRF5340_
 if BOARD_NRF5340_AUDIO_DK_NRF5340_CPUNET
 
 endif # BOARD_NRF5340_AUDIO_DK_NRF5340_CPUNET
+endif # BOARD_NRF5340_AUDIO_DK
```

Comment:
newlines between these (also the top if is not actually needed since these files will only be included when this board is used)